meta/main.yml

```
@@ -0,0 +1,8 @@
+---
+galaxy_info:
+  author: Thomas Maurice
+  description: Ansible Roie - Gitea
```

Typo, s/Roie/Role/ and you are good to go :)
```suggestion
  description: Ansible Role - Gitea
```